image/bmp.cpp

```diff
@@ -109,6 +109,8 @@ bool BitmapDecoder::loadStream(Common::SeekableReadStream &stream) {
 	_paletteColorCount = stream.readUint32LE();
 	/* uint32 colorsImportant = */ stream.readUint32LE();
 
+	stream.seek(infoSize - 40, SEEK_CUR);
```

Comment:
This addition makes sense, but it makes me wonder whether I am missing something, or if the v4 BMP (with infoSize = 108) was not working properly before. Since @dreammaster added it in 154ec750, I am guessing he needed it for something and may have tested it, so he might be able to confirm.

Reply:
@criezy 
Since pixel data is read from `subStream`, the only thing affected by this additional `stream.seek()` is palette (if present).
So it only matters in case if 8bpp BMPv4 was used. In my tests, exactly those kind of images looked broken before current PR: ![изображение](https://user-images.githubusercontent.com/5786428/124367928-89b34680-dc64-11eb-89da-07dc6f26cbb3.png)
However, you are correct, @dreammaster might have been dealing with some special resources that might be broken by this PR.

Reply:
It seems more likely that his test case only included images higher than 8bpp.

Reply:
Right. I was using the bitmaps that came with the Nuvie project, and I guess they were/are non-paletted, since they all seemed to render fine afterwards.

Reply:
I think the Nuvie BMPs are paletted non-compressed IIRC.

Reply:
@yuv422 Paletted BMPv1 (40 bytes header) images are not affected by this change as well.
Could you please point the location of Nuvie BMPs so that I could check their types?

Reply:
@lolbot-iichan The Nuvie BMPs are here:
https://github.com/nuvie/nuvie/tree/master/data
Not sure where they are located in ScummVM

Reply:
The same files are in `devtools/create_ultima/files/ultima6` in the ScummVM tree